.env.test

```diff
@@ -1,3 +1,4 @@
 RAILS_ENV=test
-RAILS_MASTER_KEY=
+# RAILS_MASTER_KEY=
 DATABASE_URL=postgres://postgres:password@localhost:5432/early_years_foundation_recovery_test
+# GOVUK_NOTIFY_API_KEY=
```

Comment:
I'm not sure having a `.env.test` is a good idea. Isn't it going to override a developer's `.env` setting when running tests?

Reply:
It depends how the dev runs them. This file will point to the containerised db if run locally.

Reply:
My understanding is that is the `.env.test` will become the dominate dotenv file when the environment is test. See https://github.com/bkeepers/dotenv#can-i-use-dotenv-in-production

Reply:
Quite right, my intention was top ensure `rspec` ran locally hit the same db as when in a container. I have taken your advice and used it as an example because rightly I shouldn't be imposing this.